OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -144,32 +144,6 @@ protected override void OnFirstRun(Actor self)
 			}
 		}
 
-		public override bool Cancel(Actor self, bool keepQueue = false)
-		{
-			if (ChildActivity == null)
-				return base.Cancel(self, keepQueue);
-
-			// Although MoveFirstHalf and MoveSecondHalf can't be interrupted,
-			// we prevent them from moving forever by removing the path.
-			if (path != null)
-				path.Clear();
-
-			// Remove queued activities
-			if (!keepQueue && NextInQueue != null)
-				NextInQueue = null;
-
-			// In current implementation, ChildActivity can be Turn, MoveFirstHalf and MoveSecondHalf.
-			// Turn may be interrupted freely while they are turning.
-			// Unlike Turn, MoveFirstHalf and MoveSecondHalf are not Interruptable, but clearing the
-			// path guarantees that they will return as soon as possible, once the actor is back in a
-			// valid position.
-			// This means that it is safe to unconditionally return true, which avoids breaking parent
-			// activities that rely on cancellation succeeding (but not necessarily immediately)
-			ChildActivity.Cancel(self, false);
-
-			return true;
-		}
-
 		public override Activity Tick(Actor self)
 		{
 			// ChildActivity is the top priority, unlike other activities.
```

Comment:
This comment has become outdated and redundant - we now have many activities that must let children run before returning NextActivity. Can you change this to `// Let child activities run so we don't end up in an odd place`?

Reply:
Fixed.